beets/ui/__init__.py

```diff
@@ -523,6 +530,13 @@ def _colorize(color, text):
         escape = COLOR_ESCAPE + "%im" % (DARK_COLORS[color] + 30)
     elif color in LIGHT_COLORS:
         escape = COLOR_ESCAPE + "%i;01m" % (LIGHT_COLORS[color] + 30)
+    elif color in MORE_COLORS:
+        # 38 == foreground color, but attributes as well as background
+        # color can be specified in the same string if desired, i.e:
+        # 38;5;196;1;3;4;5;7;48;5;220m
+        COLOR_ESCAPE = "\x1b[38;5;"
+        RESET_COLOR  = "\x1b[m"
+        escape = COLOR_ESCAPE + MORE_COLORS[color] + "m"
```

Comment:
As with the other options in this file, maybe it's easier to use string formatting instead of concatenation:

```py
escape = "\x1b[38;5;{1}m".format(color)
```